src/datastore/postgres/PgDataStore.ts

```diff
@@ -516,7 +516,13 @@ export class PgDataStore implements DataStore, ProvisioningStore {
         const row = res.rows[0];
         const config = row.config || {}; // This may not be defined.
         if (row.password && this.cryptoStore) {
-            config.password = this.cryptoStore.decrypt(row.password);
+            // NOT fatal, but really worrying.
```

Comment:
1. Why is this not fatal here?
2. Should `NedbDataStore.getIrcClientConfig` also wrap its decryption call with a try-catch?

Reply:
1. Failing to load a password shouldn't stop you from being able to use the bridge, but it will annoy the user a bit.